src/zepben/evolve/model/cim/iec61970/base/wires/power_transformer.py

```diff
@@ -294,11 +295,23 @@ class PowerTransformer(ConductingEquipment):
     result of the calculation S/Sn, where S = Load on Transformer (in VA), Sn = Transformer Nameplate Rating (in VA).
     """
 
-    def __init__(self, usage_points: List[UsagePoint] = None, equipment_containers: List[EquipmentContainer] = None,
-                 operational_restrictions: List[OperationalRestriction] = None, current_feeders: List[Feeder] = None, terminals: List[Terminal] = None,
+    asset_info: Optional[PowerTransformerInfo] = None
```

Comment:
See `Conductor` for an example of this - it should just be a property, given a more descriptive name.
e.g:
```python
@property
def power_transformer_info(self) -> PowerTransformerInfo:
    return self.asset_info
```